docs/topics/db/managers.txt

```diff
@@ -214,7 +214,7 @@ appropriate for your circumstances, you can tell Django which class to use by
 setting :attr:`Meta.base_manager_name
 <django.db.models.Options.base_manager_name>`.
 
-Manager's aren't used when querying on related models. For example, if the
```

Comment:
The apostrophe isn't correct and should be removed but it might be better to continue to say "Managers" rather than "Base managers" because neither a base manager nor any manager are used. Is it okay with you? If so, I can make that change is it looks like you deleted your Django fork.